File: media.java
import java.util.Scanner;
public class Main {
   public static void Main(String[] args){
     Scanner sc = new Scanner(System.in);
     double x, y, media;
     
     System.out.print("Digite o primeiro número: ");
     x = sc.nextDouble();
     System.out.print("Digite o segundo número: ");
     y = sc.nextDouble();
     media=(x+y)/2.0;
     System.out.printIn("Média = " + media);
     sc.close();
     
   }
}
  

File: media.cs
using System;

namespace programa {
  class Program {
    static void main(string[] args) {
      double x, y, media;
      
      Console.Write("Digite o primeiro número: ");
      x = double.Parse(Console.ReadLine());
      Console.Write("Digite o segundo número: ");
      y = double.Parse(Console.ReadLine());
      media = (x + y)/2.0;
      Console.WriteLine("Media = ", + media);
     }
    }
   } 
